File: sample/Javascript/样例.js

const { 颜色枚举, 学生元组, 中文函数 } = require('./中文包/中文模块');

/**
 * Javascript 中文命名样例
 */

function 中文函数(参数 = { 参数1: String, 参数2: String }) {
    // 枚举示例
    console.log(`喜欢的颜色: ${Object.keys(颜色枚举).find(key => 颜色枚举[key] === 颜色枚举.红色)}`);

    const 学生 = new 学生元组('李四', 17, '高二一班');
    console.log(`学生信息: ${学生.姓名}, ${学生.年龄}岁, ${学生.班级}`);

    // 处理传入参数
    if (参数.参数1) {
        console.log(`参数1: ${参数.参数1}`);
    }
}

中文函数({ 参数1: 'xxx', 参数2: 'yyy' });

File: sample/Java/中文示例/样例.java
package 中文示例;

import 中文包.颜色枚举;
import 中文包.学生元组;

/**
 * Java 中文命名样例
 */
public class 样例 {

    public static void 中文函数(String 参数1, String... 参数N) {
        // 枚举示例
        System.out.println("喜欢的颜色: " + 颜色枚举.红色.name());

        学生元组 学生 = new 学生元组("李四", 17, "高二一班");
        System.out.println("学生信息: " + 学生.toString());

        System.out.println(参数1);

        // 打印其他参数
        for (String 参数 : 参数N) {
            System.out.println(参数);
        }
    }

    public static void main(String[] args) {
        // 调用中文包中的函数
        样例.中文函数("xxx", "yyy");
    }
}
